workflows/VGP-assembly-v2/VGP-meryldb-creation-trio/meryldb-creation-trio-tests.yml

```
@@ -0,0 +1,118 @@
+- doc: Test outline for meryl_database_creation_trios.ga
+  job:
+    Collection of Paired Reads - Maternal:
+      class: Collection
+      collection_type: list:paired
+      elements:
+      - class: File
+        identifier: forward
+        localization: https://zenodo.org/record/6603774/files/paternal.fastq?download=1
+      - class: File
+        identifier: reverse
+        localization: https://zenodo.org/record/6603774/files/paternal.fastq?download=1
+    Collection of Paired Reads - Paternal:
+      class: Collection
+      collection_type: list:paired
+      elements:
+      - class: File
+        identifier: forward
+        localization: https://zenodo.org/record/6603774/files/maternal.fastq?download=1
+      - class: File
+        identifier: reverse
+        localization: https://zenodo.org/record/6603774/files/maternal.fastq?download=1
+    Pacbio Hifi reads:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: pacbio_dataset
+        localization: https://zenodo.org/record/6603774/files/child.fastq?download=1
```

```suggestion
        location: https://zenodo.org/record/6603774/files/paternal.fastq?download=1
      - class: File
        identifier: reverse
        location: https://zenodo.org/record/6603774/files/paternal.fastq?download=1
    Collection of Paired Reads - Paternal:
      class: Collection
      collection_type: list:paired
      elements:
      - class: File
        identifier: forward
        location: https://zenodo.org/record/6603774/files/maternal.fastq?download=1
      - class: File
        identifier: reverse
        location: https://zenodo.org/record/6603774/files/maternal.fastq?download=1
    Pacbio Hifi reads:
      class: Collection
      collection_type: list
      elements:
      - class: File
        identifier: pacbio_dataset
        location: https://zenodo.org/record/6603774/files/child.fastq?download=1
```

I ... don't understand how this can pass tests. `localization` isn't a known attribute.

---

Ah, the test didn't actually run. Looks like you still need a `.dockstore.yml` file here (`planemo dockstore init`).

---

Ah, ok sorry, I didn't know that this file was required for running the tests